### PR TITLE
es6-cont-let Victor K (1st submission)

### PR DESCRIPTION
**Pull request by vkalinkin**
let const

### CODE REVIEW

File: es6-const-let/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      let i = 0;
+      while (i < greetingList.length) {
+        console.log('Current greeting:', greetingList[i]);
+        i++;
+      }
+    }
+    logAllGreetings(greetings);
+
+    function getRandomGreeting(greetingList) {
+      let randomIndex = Math.floor(Math.random() * greetingList.length);
```
Comment (scbowler) [inline es6-const-let/index.html:30]: Since this won't change during the run of the function it should be `const`

**Review by scbowler — CHANGES_REQUESTED** [review]
1 update needed

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Comment by vkalinkin** [comment]
let const 2

**Pull request merged**